Userland/Libraries/LibGUI/Widget.cpp

```diff
@@ -1239,10 +1243,10 @@ bool Widget::load_from_gml_ast(NonnullRefPtr<GUI::GML::Node> ast, RefPtr<Core::O
             }
         }
 
-        return IterationDecision::Continue;
-    });
+        return {};
+    }));
 
-    return true;
+    return {};
```

Comment:
6 of one, half a dozen of the other, but instead of this return statement here, you could just `return object->try_for_each_child_object()` above

Reply:
I tried it, and I think it's a little clearer with the separate return, since the lambda is so long.